servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -32,7 +33,7 @@
  */
 public final class InodeFile extends Inode {
   private final long mBlockContainerId;
-  private final long mBlockSizeBytes;
+  private long mBlockSizeBytes;
```

Comment:
When a lineage is first added, the system creates an empty file (i.e. a place holder) with a default block size. When the job actually runs, it reinitializes the file, including the block size.